Makefile

```diff
@@ -6,20 +6,14 @@
 install-hooks: .git/hooks/pre-commit
 	@true
 
-regenerate-samples:
-	${CURDIR}/gradlew plugin:build
-	${CURDIR}/gradlew plugin:publishToMavenLocal
-	${CURDIR}/gradlew generateSwagger
-
```

Comment:
This is the main benefit from composite builds. I think this target isn't needed anymore since samples will rebuild the plugin on demand.